internal/binutils/addr2liner.go

```diff
@@ -36,6 +37,7 @@ const (
 // addr2Liner is a connection to an addr2line command for obtaining
 // address and line number information from a binary.
 type addr2Liner struct {
+	sync.Mutex
```

Comment:
Maybe make mutex a separate field like "mu sync.Mutex" like other fields? Does it need to be special?

Reply:
Done